exe/Rad.hs

```diff
@@ -0,0 +1,64 @@
+{-# LANGUAGE TypeApplications #-}
+-- | Top-level radicle CLI. Calls sub-commands.
+module Rad
+    ( main
+    ) where
+
+import           Protolude
+
+import           Data.List (isPrefixOf)
+import           Data.String (String)
+import qualified Data.Text as T
+import           System.Directory
+import qualified System.Posix.Process as Posix
+
+import           Paths_radicle (getBinDir)
+
+-- | All command filenames are prefixed with this.
+radPrefix :: String
+radPrefix = "rad-"
+
+runCommand :: [String] -> IO ()
+runCommand [] = do
+    bin <- getBinDir
+    cmds <- availableCommands [bin]
+    putStr @Text "usage: rad <command> [<args>]\n\n"
+    if null cmds then do
+        putStrLn @Text "No available rad commands."
+    else do
+        putStrLn @Text "Available rad commands:"
+        putStr $ T.unlines $ ["   " <> T.pack c | c <- cmds]
+runCommand ("version":_) =
+    putStrLn ("rad version 1.0" :: Text)
```

Comment:
You can use `version` from `Paths_radicle`. See the [cabal docs](https://cabal.readthedocs.io/en/latest/developing-packages.html?highlight=caret#accessing-the-package-version).

Reply:
Ah perfect!

Reply:
9ef9734